model/variable.cc

```diff
@@ -205,13 +205,25 @@ void VariableBase::ensureValueExists(VariableValue* vv, const std::string& nm) c
     }
 }
 
-
 string VariableBase::init() const
 {
   auto value=minsky().variableValues.find(valueId());
-  if (value!=minsky().variableValues.end())
+  if (value!=minsky().variableValues.end()) {   	
+    try
+      {
+        // set initial value of int var to init value of input to second port. for ticket 1137
+        if (!ports[0]->wires().empty())
+          if (auto i=dynamic_cast<IntOp*>(&ports[0]->wires()[0]->to()->item()))
+            if (!i->ports[2]->wires().empty()) {
+              auto vv=i->ports[2]->wires()[0]->from()->item().variableCast()->vValue();
```

Comment:
ditto.

Reply:
Probably should also check that ports.size()>2, although that should be the case with an IntOp, so maybe leave it as an assertion.